runtime/kusama/src/lib.rs

```diff
@@ -1059,6 +1060,65 @@ impl slots::Config for Runtime {
 	type WeightInfo = weights::runtime_common_slots::WeightInfo<Runtime>;
 }
 
+parameter_types! {
+	pub const CrowdloanId: PalletId = PalletId(*b"py/cfund");
+	pub const SubmissionDeposit: Balance = 100 * 100 * CENTS;
+	pub const MinContribution: Balance = 100 * CENTS;
```

Comment:
Need to adjust these number I think.

Crowdloan Creation: 10 KSM?
Min contribution: .1 KSM?